fvm/src/gas/price_list.rs

```diff
@@ -124,85 +131,91 @@ lazy_static! {
         get_randomness_per_byte: 0,
 
         block_memcpy_per_byte_cost: 0,
-        block_io_per_byte_cost: 0,
-        block_link_per_byte_cost: 1,
 
-        block_open_base: 114617,
-        block_read_base: 0,
+        block_open_base: static_milligas!(114617),
+        block_open_memret_per_byte_cost: 0,
+
+        block_link_base: static_milligas!(353640),
+        block_link_storage_per_byte_multiplier: static_milligas!(1),
+
         block_create_base: 0,
-        block_link_base: 353640,
-        block_stat: 0,
+        block_create_memret_per_byte_cost: 0,
+
+        block_read_base: 0,
+        block_stat_base: 0,
+
+        syscall_cost: 0,
+        extern_cost: 0,
 
         wasm_rules: WasmGasPrices{
             exec_instruction_cost_milli: 0,
         },
     };
 
     static ref SKYR_PRICES: PriceList = PriceList {
-        compute_gas_multiplier: 1,
         storage_gas_multiplier: 1300,
 
-        on_chain_message_compute_base: 38863,
-        on_chain_message_storage_base: 36,
-        on_chain_message_storage_per_byte: 1,
+        on_chain_message_compute_base: static_milligas!(38863),
+        on_chain_message_storage_base: static_milligas!(36),
+        on_chain_message_storage_per_byte: static_milligas!(1),
 
-        on_chain_return_value_per_byte: 1,
+        on_chain_return_value_per_byte: static_milligas!(1),
```

Comment:
Personally, I would set the `*_per_byte` storage costs to 1 instead of miligas.
They have an implicit unit of bytes, and then they get multiplied by `storage_gas_multiplier` would be `milltigas/byte`.

Reply:
Hm, it's actually implemented the other way around now. The _multiplier_ is no longer denominated in milligas, but as a raw factor. Are you good with that?

Reply:
Yeah, I’m fine with that